README.md

```diff
@@ -208,6 +208,33 @@ int total = new LengthOf(
 ).intValue();
 ```
 
+To get a set of unique elements by providing varargs:
+
+```java
+final Set<String> unique = new SetOf<String>(
+    "one",
+    "two",
+    "one",
+    "three"
+);
+```
+
+To get a set of unique elements from existing iterable:
+```java
+final Set<String> words = new SetOf<>(
+    new IterableOf<>("abc", "bcd", "abc", "ccc")
+);
+```
+
+To get sorted iterable with unique elements from existing iterable:
```

Comment:
@fanifieiev just by analogy with the previous sentences we can start this one with `To create sorted iterable`